code/__HELPERS/icons.dm

```diff
@@ -1086,23 +1086,9 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 
 GLOBAL_LIST_INIT(freon_color_matrix, list("#2E5E69", "#60A2A8", "#A1AFB1", rgb(0,0,0)))
 
-/obj/proc/make_frozen_visual()
-	// Used to make the frozen item visuals for Freon.
-	if(resistance_flags & FREEZE_PROOF)
-		return
-	if(!(obj_flags & FROZEN))
-		name = "frozen [name]"
-		add_atom_colour(GLOB.freon_color_matrix, TEMPORARY_COLOUR_PRIORITY)
-		alpha -= 25
-		obj_flags |= FROZEN
-
 //Assumes already frozed
 /obj/proc/make_unfrozen()
-	if(obj_flags & FROZEN)
-		name = replacetext(name, "frozen ", "")
-		remove_atom_colour(TEMPORARY_COLOUR_PRIORITY, GLOB.freon_color_matrix)
-		alpha += 25
-		obj_flags &= ~FROZEN
+	SEND_SIGNAL(src, COMSIG_OBJ_UNFREEZE)
```

Comment:
comment is wrong, also why does this proc exist

Reply:
only used by wet_floor component's process() it seems. wouldnt lose anything sin-wise in the current state by just making the call into the signal send but "unfreeze this obj" is relatively generic, if its ever used somewhere else we'd be doing the copypasted signal thing we all know and love